app/shared/src/commonMain/kotlin/data/source/media/MediaCacheManager.kt

```diff
@@ -169,6 +172,24 @@ abstract class MediaCacheManager(
         return false
     }
 
+    suspend fun closeAllCaches() {
+        for (storage in enabledStorages.first()) {
+            for (mediaCache in storage.listFlow.first()) {
+                mediaCache.close()
+            }
+        }
+    }
+
+    suspend fun closeAllCachesBlocking() {
+        val deferred = mutableListOf<Deferred<Unit>>()
+        for (storage in enabledStorages.first()) {
+            for (mediaCache in storage.listFlow.first()) {
+                deferred.add(backgroundScope.async { mediaCache.closeBlocking() })
+            }
+        }
+        deferred.awaitAll()
+    }
```

Comment:
backgroundScope.async 会导致泄露, `closeAllCachesBlocking` 被 cancel 后, 你启动的 job 不会被 cancel

```suggestion
    suspend fun closeAllCachesBlocking() = supervisorScope {
        for (storage in enabledStorages.first()) {
            for (mediaCache in storage.listFlow.first()) {
                launch { mediaCache.closeBlocking() }
            }
        }
    }
```